src/DelayId.cc

```diff
@@ -28,21 +28,15 @@
 DelayId::DelayId () : pool_ (0), compositeId(nullptr), markedAsNoDelay(false)
 {}
 
-DelayId::DelayId (unsigned short aPool) :
-    pool_ (aPool), compositeId (nullptr), markedAsNoDelay (false)
+DelayId::DelayId(unsigned short aPool, const DelayIdComposite::Pointer &composite) :
+    pool_(aPool), compositeId(composite), markedAsNoDelay(false)
```

Comment:
Minor naming/formatting adjustments:

```suggestion
DelayId::DelayId(unsigned short aPool, const DelayIdComposite::Pointer &aCompositeId):
    pool_(aPool), compositeId(aCompositeId), markedAsNoDelay(false)
```

Reply:
Ok.